Source/DafnyLanguageServer/Language/Symbols/ComprehensionSymbol.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+
+namespace Microsoft.Dafny.LanguageServer.Language.Symbols {
+  public class ComprehensionSymbol : ScopeSymbol, ILocalizableSymbol {
```

Comment:
I was exploring symbols when I first created this new symbol, then realized I just wanted the functionality of ScopeSymbol and then inherited it. I also realized that the custom GetDetailText does not seem to be relevant so I just removed it and now everything is a ScopeSymbol. Thanks.

Based on [all these answers](https://stackoverflow.com/questions/429470/naming-conventions-for-abstract-classes),  I don't think it's urgent to rename Symbol to SymbolBase (especially that now I added a static method there). It could even be possible to create a public API that exposes symbols down the road, so I suggest keeping the name the way they are.